samples/sample-infra/deploy-samples.bicep

```diff
@@ -0,0 +1,255 @@
+param location string = resourceGroup().location
```

Comment:
Shall we break this into three things for clarity?
1. Core Functions App
2. Services like Redis, EventHubs
3. Dapr components

Reply:
Do you mean creating 3 different files? 

or creating 3 sections in this bicep files itself?